botorch/acquisition/multi_objective/multi_output_risk_measures.py

```diff
@@ -273,10 +273,19 @@ def get_mvar_set_cpu(self, Y: Tensor) -> Tensor:
         if m != 2:  # pragma: no cover
             raise ValueError("`get_mvar_set_cpu` only supports `m=2` outcomes!")
         # Generate sets of all unique values in each output dimension.
-        unique_outcomes = [Y[:, i].unique(sorted=True).tolist()[::-1] for i in range(m)]
+        # Note that points in MVaR are bounded from above by the
+        # independent VaR of each objective. Hence, we only need to
+        # consider the unique outcomes that are less than or equal to
+        # the VaR of the independent objectives
+        var_alpha_idx = ceil(self.alpha * self.n_w) - 1
+        Y_sorted = Y.topk(Y.shape[0] - var_alpha_idx, dim=0, largest=False).values
```

Comment:
Should we pass in `sorted=True`? In my testing, the return values are sorted regardless. But PyTorch does not explicitly document what should happen when `sorted=False`. Might be safer since the algorithm depends on the values being sorted.

Reply:
The default value of the `sorted` arg is `True`, I believe? https://pytorch.org/docs/stable/generated/torch.topk.html

Reply:
That explains it. I must've completely missed that somehow.